testcafe/04_offerer.js

```diff
@@ -44,7 +44,7 @@ test('en étant sur la page de création d’une structure', async t => {
     .contains('10 PLACE JEAN JAURES - 75000 Paris')
     .click(submitButton())
     .expect(getPathname())
-    .eql('/structures')
+    .eql(HOME_URL)
 
   // Je peux créer une nouvelle structure avec un SIREN dont l'adresse n'est pas renvoyée par l'API sirene, et je suis redirigé·e vers mes structures
```

Comment:
```suggestion
  // Je peux créer une nouvelle structure avec un SIREN dont l'adresse n'est pas renvoyée par l'API sirene, et je suis redirigé·e vers la page d'accueil
```

Pour le commentaire au-dessus également